### ISSUE TITLE
cant compile from source.

### ISSUE THREAD

**Issue by UchihaSR** [Nekmo/dirhunt#82]
python version is : 3.8.5
i tried to compile from source but it is  failing. i have attached a screenshot of the error
![dirhunt_issue](https://user-images.githubusercontent.com/52357560/91556041-b2392880-e94f-11ea-8d46-e05cf3d07bd7.PNG)

**Comment by Nekmo** [comment]
`distutils` uses a invalid strict version number in your installation. This patch resolves this issue:

Current code:
```
In [6]: StrictVersion('49.6.0.post20200814') > StrictVersion('20.2')
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-6-b9e45d8783aa> in <module>
----> 1 StrictVersion('49.6.0.post20200814') > StrictVersion('20.2')

/usr/lib/python3.8/distutils/version.py in __init__(self, vstring)
     38     def __init__ (self, vstring=None):
     39         if vstring:
---> 40             self.parse(vstring)
     41 
     42     def __repr__ (self):

/usr/lib/python3.8/distutils/version.py in parse(self, vstring)
    135         match = self.version_re.match(vstring)
    136         if not match:
--> 137             raise ValueError("invalid version number '%s'" % vstring)
    138 
    139         (major, minor, patch, prerelease, prerelease_num) = \

ValueError: invalid version number '49.6.0.post20200814'
```

New code:
```
In [5]: LooseVersion('49.6.0.post20200814') > LooseVersion('20.2')
Out[5]: True
```

**Comment by Nekmo** [comment]
@UchihaSR try again using `develop` branch. This patch will be available in v0.7.1 release, thanks!
